misc/install-deps.sh

```diff
@@ -14,20 +14,25 @@ distro=$(grep "^ID=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
 case $distro in
     "ubuntu" | "debian")
         apt-get $OPT install pandoc libdw-dev libpython2.7-dev libncursesw5-dev pkg-config
-        apt-get $OPT install libcapstone-dev libluajit-5.1-dev ;;
+        apt-get $OPT install libluajit-5.1-dev || true
+        apt-get $OPT install libcapstone-dev || true ;;
```

Comment:
Is this needed for travis?  Anyway I think this change should be in a separate commit.

Reply:
Because .travis.yml is now using ubuntu 16.04, this change is not much necessary. Should I revert this change as before?

Reply:
Well, it's not important.  Just leave it.